pkgs/development/libraries/libprom/default.nix

```
@@ -0,0 +1,39 @@
+{ gccStdenv
+, fetchFromGitHub
+, cmake
+}:
+let
+  stdenv = gccStdenv;
+in
+stdenv.mkDerivation rec {
+  pname = "libprom";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "digitalocean";
+    repo = "prometheus-client-c";
+    sha256 = "0g69s24xwrv5974acshrhnp6i8rpby8c6bhz15m3d8kpgjw3cm8f";
```

Very minor nit (reordering to how most other packages organize this):
```suggestion
    owner = "digitalocean";
    repo = "prometheus-client-c";
    rev = "v${version}";
    sha256 = "0g69s24xwrv5974acshrhnp6i8rpby8c6bhz15m3d8kpgjw3cm8f";
```